### ISSUE TITLE
The glossary term "satisfies a success criterion" is not used in the WCAG with that phrasing

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag#2784]
There is the glossary term "satisfies a success criterion" that is linked in only one place, but only on the word "satisfies" when used in the [Conformance Level section](https://www.w3.org/TR/WCAG22/#cc1) in the bullet that reads:
> - For Level A conformance (the minimum level of conformance), the [Web page](https://www.w3.org/TR/WCAG22/#dfn-web-page-s) [satisfies](https://www.w3.org/TR/WCAG22/#dfn-satisfies) all the Level A Success Criteria, or a [conforming alternate version](https://www.w3.org/TR/WCAG22/#dfn-conforming-alternate-versions) is provided.

I searched the entire document and found the use of satisfy, satisfying, satisfied, satisfies which is found in various sections and criteria but none link to this definition or use the exact phrase "satisfies a success criterion".  This was also an issue in other WCAG versions but surfaced when looking at definitions in the WCAG2ICT document.

Suggest modifying the glossary term to say "satisfies/satisfied/satisfying (success criteria)" or something like that and linking other locations that use some form of that word and meaning. IMO, this is editorial as it doesn't change meaning anywhere.

**Comment by cstrobbe** [comment]
In the glossary entry, perhaps the phrase "a success criterion" should be in parentheses, so it actually reads "satisfies (a success criterion)"?
There are a few other items with parentheses in the glossary, e.g. [large scale (text)](https://www.w3.org/TR/WCAG22/#dfn-large-scale) and [relied upon (technologies that are)](https://www.w3.org/TR/WCAG22/#dfn-relied-upon).

**Comment by bruce-usab** [comment]
Sorry, I think we should leave this one alone.  I think it came from people asking the question, What does it mean to satisfy success criteria?

The definition is:
> the success criterion does not evaluate to 'false' when applied to the page

So using parenthesis buries the lede, as does the variants on 'satisfy.'

I agree that it is all a bit awkward.

**Comment by mbgower** [comment]
As per @bruce-usab's comment, the definition is at https://www.w3.org/TR/WCAG21#dfn-satisfies
> **satisfies a success criterion**
   the success criterion does not evaluate to 'false' when applied to the page

There is also a definition for conformance at https://www.w3.org/TR/WCAG22/#dfn-conform
>**conformance**
satisfying all the requirements of a given standard, guideline or specification

And then under the section on Conformance https://www.w3.org/TR/WCAG22/#cc1 we get back to the sections @maryjom cited in her ticket (which subsequent bullets listing AA and AAA):
> For Level A conformance (the minimum level of conformance), the [Web page](https://www.w3.org/TR/WCAG22/#dfn-web-page-s) [satisfies](https://www.w3.org/TR/WCAG22/#dfn-satisfies) all the Level A Success Criteria, or a [conforming alternate version](https://www.w3.org/TR/WCAG22/#dfn-conforming-alternate-versions) is provided.

So there is a fair amount of round-about referencing, but together they seem to make it clear that to conform, you must satisfy each criterion. So IMO, it could stay as it is, or we could adopt the simple recommendation of @cstrobbe :
> **satisfies (a success criterion)**
   the success criterion does not evaluate to 'false' when applied to the page

**Comment by alastc** [comment]
Discussion from the meeting:
- Whilst it is an odd formation, the effort required to update the normative text is more than the benefit from the change. (It seems to have worked ok since 2008.)
- Suggest we accept this is ok, and drop the proposed update.

@maryjom can you tolerate that?

**Comment by maryjom** [comment]
@alastc Yes, but IMO it's all editorial and not a normative change. Guess it's something to look out for in the future with definitions and phrasing to make sure things are more consistent.

**Comment by alastc** [comment]
We'll close this as we won't action on WCAG 2.x, but I've labelled it wcag.next, and it is something we can search and find.
